bundles/k3d/dev/uds-bundle.yaml

```diff
@@ -0,0 +1,143 @@
+kind: UDSBundle
+metadata:
+  name: k3d-swf-dev
+  description: A UDS bundle for deploying the UDS Software Factory on a development cluster
+  # x-release-please-start-version
+  version: 0.1.0
+  # x-release-please-end
+
+packages:
+
+  - name: dev-minio
+    repository: ghcr.io/defenseunicorns/packages/uds/dev-minio
+    ref: 0.0.1
```

Comment:
these should be de-relelease-pleased and renovated instead. (don't want to make too many GH comments)

Reply:
I dont think so? This is the version for this repo...

Reply:
you meant below this comment im assuming took me a minute...